### PR TITLE
chore(make): depreciate shelljs/make

### PR DESCRIPTION
**Pull request by ariporad**

### CODE REVIEW

File: make.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 require('./global');
 
+console.log('WARNING: shelljs/make is deprecated, and will be going away soon. You should upgrade ' +
+            'your code to stop using it. You should also submit feedback at https://git.io/vwgId.');
```
Comment (nfischer) [inline make.js]: Could we use the full URL? The shortened URL may appear as less trustworthy.

**Comment by nfischer** [comment]
@ariporad I updated the warning output to be a little more concise. I took out the URL, since I'm sure if someone has a concern, they can just post an issue and we can redirect the conversation to the original thread.

**Comment by ariporad** [comment]
LGTM!

**Comment by timburgess** [comment]
Why is make functionality being deprecated?

**Comment by nfischer** [comment]
@timburgess Please take a look at #340 and we'll continue the conversation there

**Pull request merged**